src/server/ApiRoutes/SubjectHeading.js

```diff
@@ -0,0 +1,46 @@
+import {
+  getReqParams,
+} from '../../app/utils/utils';
+import nyplApiClient from '../routes/nyplApiClient';
+import logger from '../../../logger';
+
+const nyplApiClientCall = query => nyplApiClient()
+  .then((client) => {
+    console.log(`/discovery/resources${query}`);
+    client.get(`/discovery/resources${query}`, { cache: false });
```

Comment:
should `return` this line, as in:

`     return client.get(`/discovery/resources${query}`, { cache: false });`

It's in a multi-line block, so the last line isn't returned by default (as in ruby). (Or put another way, unless you `return` something explicitly from a multi line block, there's an implied `return undefined`.) If you don't return the `Promise` returned by `client.get`, in effect the query is made, but the result is never handed off to the caller (`fetchBibs`).

Reply:
Thanks! Yes that was the problem.